product-configuration-extension/src/ProductDetailsConfigurationExtension.liquid

```diff
@@ -1,39 +1,84 @@
 {%- if flavor contains "react" -%}
 import {
-  reactExtension,
   useApi,
   Text,
+  render,
+  BlockStack,
+  Icon,
+  InlineStack,
+  Box,
+  Divider
 } from '@shopify/ui-extensions-react/admin';
 
+// Define the bundle components
+const bundleComponents = ['Burger', 'Fries'];
+
 // The target used here must match the target used in the extension's toml file (./shopify.extension.toml)
-{% if flavor contains "typescript" %}
-export default reactExtension<any>('admin.product-details.configuration.render', () => <App />);
-{% else %}
-export default reactExtension('admin.product-details.configuration.render', () => <App />);
-{% endif %}
+render('admin.product-details.configuration.render', () => <App />);
+
+// The main App component
 function App() {
+  // Uncomment the following line if you need to access the API
   {% if flavor contains "typescript" %}
-  const {extension: {target}, i18n} = useApi<'admin.product-details.configuration.render'>();
+  // const {extension: {target}, i18n} = useApi<'admin.product-details.configuration.render'>();
   {% else %}
   const {extension: {target}, i18n} = useApi();
   {% endif %}
+
+  // Render the bundle components
   return (
-    <Text>
-      {i18n.translate('welcome', {target})}
-    </Text>
+    <BlockStack gap="small">
+      {bundleComponents.flatMap((component, index) => [
+        <InlineStack gap key={component}>
+          <Box padding="none" inlineSize="10%">
+            <Icon name="ImageMajor" size="fill" />
+          </Box>
+          <Box padding="large none">
+            <BlockStack gap="none" padding="none">
+              <Text fontWeight="bold-200">
+                {component}
+              </Text>
+            </BlockStack>
+          </Box>
+        </InlineStack>,
+        index < bundleComponents.length - 1 && <Divider key={`divider-${index}`} />
+      ])}
+    </BlockStack>
```

Comment:
Can we use `<ResourceList />` and `<ResourceItem />` components? It would simplify the markup here and I think they are available in the `ui-extensions-react` library.

Reply:
Doesn't look like it exists there - https://shopify.slack.com/archives/C04K7BZDH3N/p1702483564729509?thread_ts=1702483485.885719&cid=C04K7BZDH3N